lib/Bootstrap.php

```diff
@@ -14,8 +14,10 @@
 
 namespace Pimcore;
 
+use Composer\Autoload\ClassLoader;
 use Doctrine\Common\Annotations\AnnotationReader;
-use Doctrine\Common\Annotations\AnnotationRegistry;
+use Pimcore\Loader\Autoloader\AliasMapper;
```

Comment:
This `use` statements are currently useless,  we can leave them, but then e.g. `$legacyMappingLoader = new \Pimcore\Loader\Autoloader\AliasMapper($loader);` should use only the class name.

Reply:
done